charts/eks-anywhere-packages/templates/crd.yaml

```diff
@@ -0,0 +1,488 @@
+    schema:
+      openAPIV3Schema:
+        description: PackageBundleController is the Schema for the packagebundlecontrollers
+          API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: PackageBundleControllerSpec defines the desired state of
+              PackageBundleController
+            properties:
+              activeBundle:
+                description: ActiveBundle is name of the bundle from which packages
+                  should be sourced.
+                type: string
+              logLevel:
+                description: LogLevel controls the verbosity of logging in the controller.
+                format: int32
+                type: integer
+              source:
+                description: Source of the bundle.
+                properties:
+                  registry:
+                    description: Registry is the OCR address hosting the bundle.
+                    type: string
+                  repository:
+                    description: Repository is the location of the bundle within the
+                      OCR registry.
+                    type: string
+                required:
+                - registry
+                - repository
+                type: object
+              upgradeCheckInterval:
+                description: "UpgradeCheckInterval is the time between upgrade checks.
+                  \n The format is that of time's ParseDuration."
+                type: string
+            required:
+            - source
+            type: object
+          status:
+            description: PackageBundleControllerStatus defines the observed state
+              of PackageBundleController
+            properties:
+              state:
+                description: State of the bundle controller
+                enum:
+                - ignored
+                - active
+                type: string
+            type: object
+        type: object
+    served: true
+    storage: true
+    subresources:
+      status: {}
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
do all the crds need to be on the same file? Is it possible to split them into separate files for readability?

Reply:
It is possible to split them up, was trying to reduce the total file count, but I could also create a /crds folder potentially.

Reply:
I think the /crds folder will make it easier to naviage the files. It is just my thought though.